dojo/templates/notifications/mail/sla_breach_combined.tpl

```diff
@@ -0,0 +1,68 @@
+{% load i18n %}
+{% load navigation_tags %}
+{% load display_tags %}
+<html>
+    <body>
+        {% autoescape on %}
+            <p>
+                {% trans "Hello" %} {{ user.get_full_name }},
+            </p>
+            <p>
+                {% trans "Product summary" %}:
+                <br/>
+                - {% trans "name" %}: {{product.name}}
+                <br/>
+                - {% trans "product type" %}: {{product.prod_type}}
+                <br/>
+                - {% trans "team manager" %}: {{product.team_manager}}
+                <br/>
+                - {% trans "product manager" %}: {{product.product_manager}}
+                <br/>
+                - {% trans "technical contact" %}: {{product.technical_contact}}
+            </p>
+            <p>
+                {% if breach_kind == 'breached' %}
+                  {% blocktranslate trimmed %}
+                    These security findings have breached their SLA:
+                  {% endblocktranslate %}
+                {% elif breach_kind == 'prebreach' %}
+                  {% blocktranslate trimmed %}
+                    These security findings are about to breach their SLA:
+                  {% endblocktranslate %}
+                {% elif breach_kind == 'breaching' %}
+                  {% blocktranslate trimmed %}
+                    These security findings breaching their SLA today:
+                  {% endblocktranslate %}
+                {% else %}
+                    This should not happen, check 'breach_kind' and 'kind' properties value in the source code.
+                {% endif %}
+                {% for f in findings %}
+                <br/>
+                  - <a href="{{base_url|full_url}}finding/{{f.id}}">{{f.id}}</a>: "{{f.title}}" ({{f.severity}} {% trans "severity" %}), {% trans "SLA age" %}: {{f.sla_age}}
```

Comment:
I haven't noticed it before, I will comment on it here:
- Writing hardcoded `finding/{{f.id}}` is not the best practice. Can you use `{% url 'view_finding' finding.id as finding_url %}` + `href="{{ finding_url|full_url }}"`?
- Is the ID of the finding necessary to show in the message? I would replace it with `<a href="...">"{{f.title}}" ({{f.severity}} {% trans "severity" %}), {% trans "SLA age" %}: {{f.sla_age}}</a>`
- It is better to use `<ul>` + `<li>` in stread of `<br>` + `-` (same for product summary)